Console: base routing on network / chain ID rather than DAO address

Evalir opened the pull request:
Closes #241.

- Moves routing to have a parameter to track the network (`mainnet` or `rinkeby`)
- Introduces a redirect to the DAO Selector when the network is changed on the dropdown
- Introduces a home button (the logo on the top left, just like client.aragon.org)

_pls ignore that accidental push to master :D_

Evalir committed:
console: base routing on network / chain ID rather than DAO address

bpierre commented on packages/govern-console/src/components/Header/Header.tsx:
```diff
@@ -47,7 +56,23 @@ function Header() {
+            }
+          `}
+        >
+          <img src={AragonSvg} width="36" alt="Grey Eagle Aragon logo" />
```
We should use an empty alt here, since we have the title right after it (“Govern Console”). When a logo is being used in a header, it is generally recommended to use either an empty alt (when the textual version is present) or the name of the site / app directly, rather than describing the image itself. This is to avoid screen readers repeating “Grey Eagle Aragon logo − Govern Console” every time the page loads :smile: 

Now since we are adding a button here, we need a way to describe it. We could use a `title` to describe the action, but should we move the `h1` inside the button instead? I think it would make sense to make them both clickable.

Evalir replied:
I had no idea about this for screen readers! Will do. :) Do you have any material regarding this that I could read? And yes, let's actually move it inside the button. Looking back, it feels weird that you can click the logo but not the title. 😄 

Done in https://github.com/aragon/govern/pull/245/commits/05c49a7dafb23387391101e507c48fb78b275a35

bpierre replied:
> Do you have any material regarding this that I could read?

Of course!

The WAI tutorial about images describes a case similar to this one: https://www.w3.org/WAI/tutorials/images/functional/#logo-image-within-link-text

Except in our case, the text doesn’t describe the actual action… but a click on a logo is generally expected to have the “go to home” behavior, so I usually omit it. This is also [what the WebAIM website recommends](https://webaim.org/techniques/alttext/#logos):

> Developers will often identify the image as being to the home page (alt="Acme Company home page), though if the image is consistently at the beginning of the page and the alternative text is appropriate, this additional information should not be needed.

bpierre reviewed (approved):
LGTM! Just left a comment about the `alt=""` attribute for the logo.

Evalir committed:
console: move title inside button and remove alt prop for screen readers

Quoting Pierre:
"We should use an empty alt here, since we have the title right after it
(“Govern Console”). When a logo is being used in a header, it is
generally recommended to use either an empty alt (when the textual
version is present) or the name of the site / app directly, rather than
describing the image itself. This is to avoid screen readers repeating
“Grey Eagle Aragon logo − Govern Console” every time the page loads 😄

Now since we are adding a button here, we need a way to describe it. We
could use a title to describe the action, but should we move the h1
inside the button instead? I think it would make sense to make them both
clickable.
"
